code/script.js

```diff
@@ -36,12 +44,183 @@ const greeting = () => {
+          <option id="select">Hey Jude by The Beatles</option>
+        </select>
+      ` 
+      const select = document.getElementById("select");
+      select.addEventListener("change", () =>
+      greatChoice(select.value))
+    }
+  })
+
+  document.getElementById("option3").addEventListener("click", () => {
+      showMessage("I see that you are angry today😡. Here's a list that suits your mood. Go ahead and choose one! 👇", "bot")
+      inpt = () => {
+        inputWrapper.innerHTML=`
+          <select id="select" name="songs">
+            <option value selected disabled>Select a song</option>
+            <option id="">In the end by Linkin Park</option>
+            <option id="">Basket case by Green Day</option>
+            <option id="">Break Stuff by Limp Bizkit</option>
+         </select>
+        `
+        const select = document.getElementById("select");
+        select.addEventListener("change", () =>
+        greatChoice(select.value))
+      }   
+   })
+
+  setTimeout(() => {
+    inpt();
+  }, 2000)
+  
+}
+
+//Question 3
+const greatChoice = () => {
+  showMessage(`Great choice! I love that song ❤ Do you want to play it now or later`, 'bot')
+  setTimeout(() => greatChoice(value, 'bot'), 2000)
+  inputWrapper.innerHTML = `
+  <button id="now">NOW</button>
+  <button id="later">LATER</button>
+  `
+   document.getElementById("now").addEventListener("click", () => {
+        showMessage("Ok, let's listen to it right away! Enjoy your song! Goodbye!", "bot")
+        inpt = () => {
+        }
+    })
+  
+    document.getElementById("later").addEventListener("click", () => { 
+        showMessage("Later it is! Enjoy your song! Goodbye!", "bot")
+        inpt = () => {
+          const select = document.getElementById("select");
+          select.addEventListener("change", () =>
+          greatChoice(select.value))
+        }
+    })
+      setTimeout(() => {
+      inpt();
+    }, 2000)
+  }
+   
+/*//Question 3
+const greatChoice = () => {
+showMessage(`Great choice! I love that song ❤ Do you want to play it now or later?`, 'bot')
+setTimeout(() => greatChoice(value, 'bot'), 2000)
+  inputWrapper.innerHTML = `
+    <button id="now">NOW</button>
+    <button id="later">LATER</button>
+`
+
+
+}
+
+//adding a new function in case it works
+const whenToplaySong = () => {
+  inputWrapper.innerHTML = `
+  <button id="now">NOW</button>
+  <button id="later">LATER</button>
+`
+  document.getElementById("now");
+  showMessage("Ok, let's listen to it right away! Enjoy your song! Goodbye!", "bot")
+
+  document.getElementById("later");
+  showMessage("Later it is! Enjoy your song! Goodbye!", "bot")
+  } 
+
+*/
+
 // Set up your eventlisteners here
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const value = document.getElementById('name-input').value;
+  showMessage(value, 'user')
+});
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  showMessage(value, 'bot')
+});
```

Comment:
You created 3 listeners for the same action and object and this is probably why your inputs do not always show correctly.